src/tests/test_array.py

```diff
@@ -143,7 +143,7 @@ def testArrayContains(self):
 
         self.assertFalse(5 in items)    # "H:\Python27\Lib\unittest\case.py", line 592, in deprecated_func,
         self.assertFalse(-1 in items)   #TypeError: int() argument must be a string or a number, not 'NoneType'
-        self.assertFalse(None in items) # which threw ^ here which is a little odd.
```

Comment:
looks like this test used to break at some point. I also broke it in PTVS before these merges:

https://github.com/pythonnet/pythonnet/issues/122